### PR TITLE
Popup window

### PR DESCRIPTION
**Pull request by rashedarman**
- [x] When the user clicks (or taps) the button to check project details, the popup with details about the project appears.
- [x] When the user clicks (or taps) the close (X) button, the popup disappears.
- [x] Popup modal is responsive

### CODE REVIEW

File: modal.js [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+  {
+    title: 'Keeping track of hundreds of components 6',
+    description:
+      'Lorem ipsum dolor sit amet consectetur adipisicing elit. Perferendis temporibus sint fugiat blanditiis, quo aperiam, vel maxime rerum sunt dignissimos, dolorem tenetur id! Exercitationem magnam unde veniam id molestiae totam! ',
+    image: './assets/popup-image.png',
+    technologies: ['Ruby on rails', 'css', 'JavaScript'],
+    live: '#',
+    source: '#',
+  },
+];
```
Comment (Tadesse-Alemayehu) [inline modal.js:56]: - [] Dear @rashedarman, Good job on the design. Although for all the cards in your project please make sure it is not statically typed on the HTML file. This array of projects should be the one to be displayed on your HTML. Please make sure to remove the static HTML code and dynamically render this list of projects. 

## this code should be replaced by a dynamic JavaScript.
https://github.com/rashedarman/portfolio/blob/4330d9e42a2327ad3cd20121810d3d1536c629c6/index.html#L84-L163

File: index.html [hunk 2]
```diff
@@ -273,6 +276,30 @@ <h3 class="heading-primary">
...
+            See live
+            <i class="fa-solid fa-link"></i>
+          </a>
+          <a class="btn-primary">
+            See source
+            <i class="fa-brands fa-github fa-lg"></i>
+          </a>
+        </div>
+      </div>
+    </div>
```
Comment (Tadesse-Alemayehu) [inline index.html:301]: - [] Although your modal is looking great, Its content should not be filled with static text. This content should be rendered from the JavaScript array you have created already. (when the user click on the project, the popup should be populate with the detail of that project)

**Review by Tadesse-Alemayehu — CHANGES_REQUESTED** [review]
### Hi @rashedarman 

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
EquatorialVictoriousBlackpanther-size_restricted

## Highlights
 - [ ] Good job overall following the template design 💯
 - [ ]  Correct GitHub Flow 👍🏾
 - [ ]  Good commit history 🥇
 - [ ]  Good PR title/description 💯
 - [ ]  All linters passed 🥇
 
## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by algerina — APPROVED** [review]
Hi @rashedarman , 
Good job so far! 

May I highlight the following:

- The repo looks professional.:heavy_check_mark:
- There are no linter errors:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:
- Your project looks awesome:heavy_check_mark:

## STATUS: APPROVED :muscle::clap:

Congratulations! :tada:
Your project is complete!  It's time to merge it :shipit:
Great work on implementing the changes requested by the previous TSEs


 
 ## Optional suggestions
 
 - Please check the comment under the review.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread and tag me @algerina if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Pull request merged**